src/llmcompressor/utils/helpers.py

```diff
@@ -1087,31 +1089,42 @@ def __exit__(self, _exc_type, _exc_val, _exc_tb):
 
 
 @contextlib.contextmanager
-def DisableQuantization(model: torch.nn.Module):
+def DisableQuantization(module: torch.nn.Module):
     """
     Disable quantization from QuantizationModifier
     """
     try:
-        model.apply(disable_quantization)
+        module.apply(disable_quantization)
         yield
     finally:
-        model.apply(enable_quantization)
+        module.apply(enable_quantization)
 
 
 @contextlib.contextmanager
-def calibration_forward_context(model: torch.nn.Module):
+def eval_context(module: torch.nn.Module):
+    restore_value = module.training
+    try:
+        module.train(False)  # equivalent to eval()
+        yield
+
+    finally:
+        module.train(restore_value)
+
+
+@contextlib.contextmanager
+def calibration_forward_context(model: PreTrainedModel):
     """
     Context in which all calibration forward passes should occur.
 
     - Remove gradient calculations
     - Disable the KV cache
     - Disable quantization from QuantizationModifier
+    - Disable train mode and enable eval mode
```

Comment:
Since we're updating this, can we update the comment about `Disable quantization from QuantizationModifier` to disable QDQ during forward passes? Slightly more accurate in what it disables.

Reply:
Good call